### ISSUE TITLE
onOpen of SelectableLinkify is never called

### ISSUE THREAD

**Issue by lukelyyeung** [Cretezy/flutter_linkify#31]
**SelectableLinkify** 's onOpen is never called. Maybe related to https://github.com/flutter/flutter/issues/43494

Platform: iOS 13.3

Reproducible code

```
import 'package:flutter/material.dart';
import 'package:flutter_linkify/flutter_linkify.dart';
import 'dart:async';

import 'package:url_launcher/url_launcher.dart';

void main() => runApp(new LinkifyExample());

class LinkifyExample extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return new MaterialApp(
      debugShowCheckedModeBanner: false,
      title: 'flutter_linkify example',
      home: Scaffold(
        appBar: AppBar(
          title: Text('flutter_linkify example'),
        ),
        body: Center(
          child: SelectableLinkify(
            onOpen: _onOpen,
            text: "https://github.com",
          ),
        ),
      ),
    );
  }

  Future<void> _onOpen(LinkableElement link) async {
    if (await canLaunch(link.url)) {
      await launch(link.url);
    } else {
      throw 'Could not launch $link';
    }
  }
}
```

flutter doctor result
```
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Mac OS X 10.14.6 18G95, locale en-HK)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/setup/#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, set ANDROID_HOME to that location.
      You may also want to add it to your PATH environment variable.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
[!] Android Studio (not installed)
[✓] Connected device (2 available)

! Doctor found issues in 2 categories.
```

**Comment by Cretezy** [comment]
Yep this looks valid. I'll take a look at it later today!

**Comment by AndroidNils** [comment]
Same issue.

**Comment by Cretezy** [comment]
Unfortunately, this seems to be a Flutter bug. I'll keep this ticket open until it is resolved upstream.

https://github.com/flutter/flutter/issues/43494

**Comment by da-revo** [comment]
Same problem. Good Luck

**Comment by Sameerkash** [comment]
[``](https://github.com/flutter/flutter/issues/43494#issuecomment-584497405)

this seems to have the solution to the issue.

**Comment by hkndzdr** [comment]
> [#28 43494](https://github.com/flutter/flutter/issues/43494#issuecomment-584497405)
> 
> this seems to have the solution to the issue.

It's not the right solution. It has a link feature in texts where there is no link feature. Multitext is everywhere clickable and has link feature. This is not what is desired.

**Comment by dvird** [comment]
?

**Comment by rajathk** [comment]
@Cretezy Any update on this please?

**Comment by Cretezy** [comment]
This has been fixed in Flutter, see linked thread. Please run flutter upgrade. It might only be in master/dev so far, so hold on a bit longer!

**Comment by rajathk** [comment]
Thanks @Cretezy. Just verified the fix in dev channel version 1.18.0-10.0.pre.
